transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -50,27 +50,18 @@
   not ideal, it was decided to use the different 'version' capabilities in KMS
   to isolate production from the other two envirments. Specificly: rucontenttype
   is a three level tree in SIT, but not PROD. This process is only needed durring
-  the transition and should be deleted once in production."
- )
+  the transition and should be deleted once in production.
 
-(def production?
-  "Return true if CMR is not currently in production."
-  (= "prod" (System/getenv "ENVIRONMENT")))
+  To override the scheme KMS settings, set the config kms-scheme-override-json to:
+  {\"platforms\":\"static\",
+   \"mime-type\":\"mimetype?format=csv&version=special\"}"
+ )
 
 (defn- scheme-overrides
-  "Allow for any KMS resource URL to be overriden by the system envirnmental
-   variables. The ENV is assumed to contain JSON"
-  ;; assume the value comes from an ENV
-  ([] (scheme-overrides (config/kms-scheme-override-json)))
-
-  ;; parse and return a map
-  ([json-string] (json/parse-string json-string true)))
-
-(comment
- ;; testing with an env is not convenient, use this to diretly test what the
- ;; function would do in the real world with a valid value
- (scheme-overrides (str "{\"platforms\":\"file://frozen/platforms.csv\","
-                        "\"mime-type\":\"mimetype?format=csv&version=special\"}")))
+  "CMR will Allow for any KMS resource URL to be overriden by a config
```

Comment:
Update the comment to say like "Returns the KMS schema override map as configured in CMR_KMS_SCHEMA_OVERRIDE_JSON."